components/ShareWithFriends/index.tsx

```diff
@@ -0,0 +1,22 @@
+import CopyTextIcon from '@components/CopyTextIcon';
+import { useEffect, useState } from 'react';
+
+import styles from './styles.module.css';
+
+export default function ShareWithFriends(){
+    const [url, setUrl] = useState('');
+
+    useEffect(() => {
+        setUrl(window.location.href);
+    }, [])
+
+    return (
+        <div className={styles.container}>
+            <h2>Compartilhe com seus amigos</h2>
+            <div className={styles['copy-container']}>
+                <span>{url}</span>
+                <CopyTextIcon text={url} />
```

Comment:
Pensando aqui, será que não daria pra fazer todo o "textarea" ser clicável? Pra mim parece que faz mais sentido do que só o botãozin

Reply:
Ah, sim! Eu gostei do estilo visual, to comentando só sobre, quando a pessoa clicar em qualquer lugar da textarea ele já copiar, entende?

Sobre o bug, eu testei no Chrome no iOS.

Reply:
Entendi sim, irei corrigir essa falha no icone e alterar a área disponível para clique ao copiar assim qie possível.

Reply:
Atualizei a área disponível para cópia do texto e tentei arrumar o problema com o icone ( se puder me dar um feedback sobre este, pelo DevTools não aparece o erro, apenas nos dispositivos mesmo ).